### DISCUSSION TITLE
[V5] server.ts file changes

### DISCUSSION THREAD

**Discussion by vkaelin** [adonisjs/core#1868] [Help] [answered]
Hi! 
I recently find out that if you create a new Adonis 5 app from scratch the code inside the ``server.ts`` file is not the same anymore as in older projects. Should I update it as well ? I ask it there because I didn't see anything about that inside the October update instructions.

The changes are the same in the ``ace`` file at the root of the project.

Old version:
```ts
import 'reflect-metadata'
import sourceMapSupport from 'source-map-support'
import { Ignitor } from '@adonisjs/core/build/src/Ignitor'

sourceMapSupport.install({ handleUncaughtExceptions: false })

new Ignitor(__dirname)
  .httpServer()
  .start()
  .catch(console.error)
```

New version:
```ts
import 'reflect-metadata'
import { Ignitor } from '@adonisjs/core/build/standalone'

new Ignitor(__dirname).httpServer().start()
```

**Comment by thetutlage** [comment, marked as answer]
I think `source-map-support` should still be installed so that the error stack trace in production build can point to the source files. I will fix the generator code for the new release. In other words, the old file is correct

**Reply by vkaelin** [reply]
Good to know, thanks !
